internal/signature/sign.go

```diff
@@ -209,3 +242,39 @@ func matchSAN(cert *x509.Certificate, name, email string) bool {
 
 	return false
 }
+
+func attachRekorLogEntry(ctx context.Context, sd *cms.SignedData, cert *x509.Certificate, addr string) (*models.LogEntryAnon, error) {
```

Comment:
Nit: maybe `createAndAttachRekorLogEntry`?

Attach makes me thing it already exists